### ISSUE TITLE
Changing Trigger has wrong behaviour

### ISSUE THREAD

**Issue by cosimomeli** [mibexsoftware/bamboo-plan-dsl-plugin#51]
Plugin version 1.8.2 - 1.9.0 | Bamboo version 6.0.3

I tried to replace a Remote Trigger with a Polling Trigger, but after execution I found no polling trigger but just the remote one with empty "Trigger IP addresses" (previously not empty).

In detail, I changed:
```
plan.triggers.with {
            remote() {
                repositories repos
                ipAddresses(*config.trigger_ips)
            }
}
```

with:

```
plan.triggers.with {
            polling() {
                repositories repos
                periodically {
                    pollingFrequencyInSecs config.polling_trigger_frequency
                }
            }
}
```

The execution gives no errors, but just 'polling is deprecated' warnings.
Tried both Disable and Delete for "Action for not-referenced Bamboo objects".
Is this related to the issue #50 ?

**Comment by mrueegg** [comment]
Thanks for the detailed bug report. This is indeed related to #50. I have a bug fix ready and will release a new version very soon. I'll report back here when it is out :-)

**Comment by mrueegg** [comment]
Bug fix shipped with release 1.9.1. Thanks again for reporting this!
